home/tests/test_models.py

```diff
@@ -895,23 +895,64 @@ def test_clean_text_char_limit(self) -> None:
 
 @pytest.mark.django_db
 class TestWhatsAppTemplate:
-    def test_variables_are_numeric(self) -> None:
+    @override_settings(WHATSAPP_ALLOW_NAMED_VARIABLES=False)
+    def test_positional_variables_are_numeric(self) -> None:
         """
         Template variables are numeric.
         """
```

Comment:
This tests that non-numeric positional variable are invalid, not that numeric ones are valid. Can you update the name and docstring, please?

Reply:
Updated